Update/omot_017.txt

```diff
@@ -3290,7 +3290,7 @@ void main()
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	if (GetGlobalFlag(GADVMode)) { OutputLine(NULL, "", NULL, "<size=-2>", Line_Continue); }
 	OutputLine(NULL, "まず、富竹に鷹野を疑わせることが第一。そして、その派生として真相の一部でもつかんでもらえたら、今後の展開が有利にも運ぶうえ富竹の生還も、可能性が高まるだろう。",
-			NULL, "The first step is to make Tomitake suspicious of Takano. Then, if he's able to arrive at the truth as part of this charade, Tomitake will prove advantageous in the future. And it might even increase his chances of survival.", Line_Normal);
+			NULL, "The first step is to make Tomitake suspicious of Takano. Then, if he's able to arrive at even a part the truth thanks to this charade, Tomitake will prove advantageous in the future. And it might even increase his chances of survival.", Line_Normal);
```

Comment:
```suggestion
			NULL, "The first step is to make Tomitake suspicious of Takano. Then, if he's able to arrive at even part the truth thanks to this charade, Tomitake will prove advantageous in the future. And it might even increase his chances of survival.", Line_Normal);
```